{"name": "Cannot update the \"point scale\" of project strange values -- @1.9 ", "status": "passed", "steps": [{"name": "Given I set up a \"PUT\" request to \"/projects/{proj_id}\" endpoint", "status": "passed", "start": 1556562447735, "stop": 1556562447736}, {"name": "And I set up the data", "status": "passed", "attachments": [{"name": ".text", "source": "911e4ee8-09ce-44c7-bd90-239ae207153c-attachment.txt", "type": "text/plain"}], "start": 1556562447736, "stop": 1556562447740}, {"name": "When I send the request", "status": "passed", "start": 1556562447740, "stop": 1556562448269}, {"name": "Then I get a \"400\" status code as response", "status": "passed", "start": 1556562448270, "stop": 1556562448271}, {"name": "And I verify the general_problem of error is: \"Projects require a point scale., This is not a valid custom point scale.\"", "status": "passed", "start": 1556562448271, "stop": 1556562448272}], "parameters": [{"name": "values", "value": "null"}, {"name": "message", "value": "Projects require a point scale., This is not a valid custom point scale."}], "start": 1556562447733, "stop": 1556562448273, "uuid": "a6906d82-7699-4843-91d2-d675cc0b3e32", "historyId": "4a2b2e2e431e358c6e742b272c2e1649", "labels": [{"name": "tag", "value": "negative"}, {"name": "tag", "value": "project"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "Functional Test Negative Cases of Update Projects"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}